en/forms.texy

```diff
@@ -602,6 +602,13 @@ $form->addText('email', 'E-mail:')->setDisabled(TRUE);
 
 The input cannot be written to and its value won't be returned by `getValues()`.
 
+[.tip]
+If we need readonly input (input will be disabled in HTML but its value will be shown), we disable input for first and set a value afterwards. The reason is `setDisabled()` resets value of input.
```

Comment:
If we need a read-only input (the input will rendered as disabled but its value will be shown), we disable the input first and set the value afterwards. The reason is that `setDisabled()` method resets the value of the input.